client/src/helpers/api.js

```diff
@@ -117,3 +117,16 @@ export const deleteTopic = async topic_id => {
 export const addTopic = async (title, lesson_id) => {
   return await instance.post("/api/topics", { title, lesson_id });
 };
+
+export const getUsers = async () => {
+  return await instance.get("/api/users").then(res => res.data);
```

Comment:
I think you need to use `.then` when you use `async/await`.

Reply:
ok but How to take res.data I don't know

Reply:
You can say `const res = await instance.get("...")`, and in next line `return res.data`. However, I don't think you need `res.data` in this function. Test it inside your component by logging in console and see what it shows with and without `res.data`. Also, in component you can do the same when you `await` you might need `res.data`. You can have a look.

Reply:
I need res.data. Loic told me about this if you need res.data you can do in api.js this is good coding. he said.

Reply:
Have you tried what I said? If not, fair enough.

Reply:
I looked again I don't need getUsers . I cleaned that it is still working.